docs/ReleaseNotes.md

```diff
@@ -1,6 +1,23 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.2
+Acuminator 2.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 2.2, the following enhancements have been implemented:
+ - The [PX1019](diagnostics/PX1019.md) diagnostic has been added to check for DAC fields of non-strings types with the `AutoNumberAttribute` attribute.
+ - The [PX1020](diagnostics/PX1020.md) diagnostic has been added to check if the length of a string DAC field with the `AutoNumberAttribute` attribute is sufficient.
+ - The [PX1022](diagnostics/PX1022.md) diagnostic has been added to check if a DAC or graph extension is declared as a public class.
+ - The context menu of Acuminator has been updated: Sort commands have been divided in the context menu, active sort indicator has been added, and menu control templates have been redesigned to integrate the Visual Studio theme.
+ - The following complementary information about DAC fields is displayed in a separate column: the field type, an indicator of whether the field is an identity field, and an indicator of whether the field is bound or unbound.
```

Comment:
I would add "information  about DAC fields is displayed in Code Map in  a separate column"